src/lib/requireAuth.ts

```diff
@@ -9,7 +9,6 @@ import { SessionUser } from 'types'
 
 type WithAuthServerSidePropsResult = GetServerSidePropsResult<{
   user: SessionUser
-  [key: string]: any
 }>
```

Comment:
This was causing a lint error so I removed it. Let me know if I'm missing something here, but this seems like a catch-all type to me, and I don't think it's good practice to have something like this. I'm thinking it was maybe added in early in the project when there were more unknowns, but I think we can be more explicit now.